cosmos/keybase.go

```diff
@@ -23,11 +24,23 @@ func NewKeybase(dir string) (*Keybase, error) {
 }
 
 // NewMnemonic returns a new mnemonic phrase.
-func (kn *Keybase) NewMnemonic() (string, error) {
+func (kb *Keybase) NewMnemonic() (string, error) {
 	// read entropy seed straight from crypto.Rand and convert to mnemonic
 	entropySeed, err := bip39.NewEntropy(mnemonicEntropySize)
 	if err != nil {
 		return "", err
 	}
 	return bip39.NewMnemonic(entropySeed)
 }
+
+// Exist checks if the account exists.
+func (kb *Keybase) Exist(name string) (bool, error) {
+	_, err := kb.Get(name)
+	if keyerror.IsErrKeyNotFound(err) {
+		return false, nil
+	}
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+}
```

Comment:
return !keyerror.IsErrKeyNotFound(err), err

or just remove the wrapper

Reply:
You suggestion doesn't work as `err` still contain the `IsErrKeyNotFound`.
The best I can do is:
```go
	exist := !keyerror.IsErrKeyNotFound(err)
	if err != nil && exist {
		return exist, err
	}
	return exist, nil
```
and it's not so much better.